src/Stratis.Bitcoin.Features.Wallet/Controllers/WalletController.cs

```diff
@@ -472,32 +474,42 @@ public IActionResult GetHistory([FromQuery] WalletHistoryRequest request)
                         HdAddress address = item.Address;
 
                         // First we look for staking transaction as they require special attention.
-                        // A staking transaction spends one of our inputs into 2 outputs or more, paid to the same address.
+                        // A staking transaction spends some of our inputs into 2 outputs or more, paid to the same address.
                         if (transaction.SpendingDetails?.IsCoinStake != null && transaction.SpendingDetails.IsCoinStake.Value)
```

Comment:
This can be shortened to: `if (transaction.SpendingDetails?.IsCoinStake == true)`